doc/modules/linear_model.rst

```diff
@@ -786,8 +786,12 @@ non-smooth `penalty="l1"`. This is therefore the solver of choice for sparse
 multinomial logistic regression. It is also the only solver that supports
 `penalty="elasticnet"`.
 
-In a nutshell, the following table summarizes the penalties supported by
-each solver:
+The "lbfgs" is an optimization algorithm that approximates the Broyden–Fletcher–Goldfarb–Shanno algorithm [8]_,
+which belongs to quasi-Newton methods. It performs the best for small datasets
+than other solvers that use Stochastic Gradient descent. The "lbfgs" solver is recommended for use for small data-sets
```

Comment:
Please try to keep under 80 chars per line.

Reply:
OK let me update.